content/_data/changelogs/lts.yml

```diff
@@ -0,0 +1,26 @@
+- version: 2.32.2
+  date: 2017-02-01
+  changes:
+    - type: security
+      message: Important security fixes
+      references:
+        - url: https://jenkins.io/security/advisory/2017-02-01/
+          title: security advisory
+    - type: major rfe
+      message: Support displaying of warnings from the update site in the plugin manager and in administrative monitors
+      references:
+        - issue: 40494
+        - url: https://jenkins.io/blog/2017/01/10/security-warnings/
+          title: announcement blog post
+    - type: bug
+      message: Correctly state that Jenkins will refuse to load plugins whose dependencies are not satisfied in plugin manager
+      issue: 40666
+    - type: bug
+      message: The <tt>install-plugin</tt> CLI command now correctly installs plugins when multiple file arguments are specified
+      pull: 2674
+    - type: bug
+      message: Properties were _not_ passed to Maven command by Maven build step when the Inject Build Variables flag was not set
+      issue: 39268
```

Comment:
I grepped the weekly changelog and found 0 referenced to other projects JIRAs. So I guess this is not such a problem after all.